Tests/UITests/TestApp/Views/HelperViews/CustomToggleView.swift

```diff
@@ -0,0 +1,39 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct CustomToggleView: View {
+    var text: String
+    @Binding var condition: Bool
+    
```

Comment:
Very minor change that also affects some other files: We generally keep two empty lines between the import statements and the file content as well as between different types in a Swift file.
Within a type, we use two empty lines to distance different element type in a view, e.g. stored properties have a two line space that separates them from computed properties:
```suggestion
    
    
```